rabbitmq/check.py

```diff
@@ -353,7 +353,7 @@ def get_connections_stat(self, instance, base_url, object_type, vhosts, custom_t
         for conn in data:
             if conn['vhost'] in vhosts:
                 stats[conn['vhost']] += 1
-                connection_states[conn['state']] += 1
+                connection_states[conn.get('state', 'direct')] += 1
```

Comment:
Why `direct` as a default value ?
If `conn` don't have any state `unknown` seems better in my opinion.